docs/operations.md

```diff
@@ -0,0 +1,346 @@
+# Operations
+
+This is the work in progress operations guide for showing information,
+which are relevant for production use.
+
+Skipper is proven to scale with number of routes beyond 200.000 routes
+running with peaks to 45.000 http requests per second.
+
+## Connection Options
+
+Skipper's connection options are allowing you to set Go's [http.Server](https://golang.org/pkg/net/http/#Server)
+Options on the client side and [http.Transport](https://golang.org/pkg/net/http/#Transport) on the backend side.
+
+There is a [blog post about net http
+timeouts](https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/),
+that I highly recommend if you want to understand the impact of these
```

Comment:
in documentation, it is recommended to avoid using first/second person singular form. Maybe:

"It is recommended to read [this blog post about net http timeouts](https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/) in order to better understand the impact of these settings."

Reply:
thanks fixed